app/src/main/java/com/futurice/freesound/feature/search/SearchFragment.java

```diff
@@ -92,18 +96,18 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_search, container, false);
+        View view = inflater.inflate(R.layout.fragment_search, container, false);
```

Comment:
I know the ButterKnife docs say to do it here, but it feels odd to be doing the binding in a place where the view is being created: `onCreateView` rather than in the callback indicating the View is ready: `onViewCreated`. If there's no hard requirement to do it in `onCreateView`, I would prefer if the binding was deferred until `onViewCreated` for the sake of mixing together the two responsibilities.